editor/modes/visual-editor/block.js

```diff
@@ -200,18 +207,27 @@ class VisualEditorBlock extends wp.element.Component {
 			>
 				{ ( showUI || isHovered ) && <BlockMover uid={ block.uid } /> }
 				{ showUI &&
-					<div className="editor-visual-editor__block-controls">
-						<BlockSwitcher uid={ block.uid } />
-						{ !! settings.controls && (
-							<Toolbar
-								controls={ settings.controls.map( ( control ) => ( {
-									...control,
-									onClick: () => control.onClick( block.attributes, this.setAttributes ),
-									isActive: control.isActive ? control.isActive( block.attributes ) : false,
-								} ) ) } />
-						) }
-						<Slot name="Formatting.Toolbar" />
-					</div>
+					<CSSTransitionGroup
+						transitionName={ { appear: 'appear-animation', appearActive: 'is-appearing' } }
```

Comment:
>Even if we're not using it, it's required to avoid a JS error.

Could we just use a string, like `transitionName="is-appearing"`. Wouldn't that give us a `is-appearing-active` modifier? Seems reasonable enough naming to me.